### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by nfnt**
* Updated ToCs
* Some markdown linting
* Update some outdated descriptions
* Add sections for failure handling

### CODE REVIEW

File: docs/managing.md [hunk 1]
```diff
@@ -78,6 +83,76 @@ configurable settings.
...
+keyspaces with a replication factor of ONE, or the data of the failed Cassandra
+node will be lost.
+
+#### Node eviction
+
+Evicting a Cassandra node is similar to Failure recovery described above. The
+recovery controller will automate certain steps. The main difference is that
+during node eviction the Kubernetes node should stay available, i.e. other pods
+on that node shouldn’t get evicted. To evict a Cassandra node, first cordon or
+taint the Kubernetes node the Cassandra node is running on. This ensures that
```
Comment (ANeumann82) [inline docs/managing.md:133]: We can now also add a section to describe that KUDO Cassandra supports the cordon for Cassandra only.

If you want to cordon only Cassandra from a specific node, add a label `kudo-cassandra/cordon=true` to that node. KUDO Cassandra pods will not be scheduled to Kubernetes nodes with this label present.

File: docs/multidatacenter.md [hunk 2]
```diff
@@ -14,9 +14,13 @@ rack awareness.
 
 ## Kubernetes cluster prerequisites
 
-At this time, KUDO Cassandra needs a single Kubernetes cluster spanning all the
-datacenters. A Cassandra cluster running on two or more Kubernetes clusters is
-not supported at the moment
+### Naming
+
+Cassandra datacenters can either run in a single Kubernetes cluster that is
```
Comment (ANeumann82) [inline docs/multidatacenter.md:19]:
```suggestion
Cassandra clusters can either run in a single Kubernetes cluster that is
```
?

File: docs/multidatacenter.md [hunk 3]
```diff
@@ -205,8 +214,7 @@ in your cluster as you use in the NODE_TOPOLOGY definition.
 
 ### Full list of required parameters
 
-```
-    ENDPOINT_SNITCH=GossipingPropertyFileSnitch
-    NODE_ANTI_AFFINITY=true
-    NODE_TOPOLOGY=<the cluster topology>
+```yaml
+ENDPOINT_SNITCH=GossipingPropertyFileSnitch NODE_ANTI_AFFINITY=true
+NODE_TOPOLOGY=<the cluster topology>
```
Comment (ANeumann82) [inline docs/multidatacenter.md:219]:
```suggestion
```yaml
ENDPOINT_SNITCH=GossipingPropertyFileSnitch 
NODE_ANTI_AFFINITY=true
NODE_TOPOLOGY=<the cluster topology>
```

File: docs/managing.md (hunk 1 above)
Comment (porridge) [reply docs/managing.md:133]: `s/the cordon for Cassandra only/the cordon specific to KUDO-Cassandra/`

File: docs/multidatacenter.md (hunk 2 above)
Comment (nfnt) [reply docs/multidatacenter.md:19]: Mutiple Datacenters form a single Cassandra cluster. At least that's what I'm trying to say here. I'll add another sentence to distinguish between Cassandra cluster and datacenter.

File: docs/multidatacenter.md (hunk 3 above)
Comment (nfnt) [reply docs/multidatacenter.md:219]: Oops, how did that happen?

Comment (nfnt) [reply docs/multidatacenter.md:220]: It's the file formatter for YAML. Just noticed that this isn't actually YAML, changing this to text formatting here and in other places.

**Pull request merged**